app/views/shop/products/_summary.html.haml

```diff
@@ -1,5 +1,7 @@
 .product-thumb
   %a{"ng-click" => "triggerProductModal()"}
+    %span.product-thumb__bulk-label{"ng-if" => "::product.group_buy"}
+      = t("products_bulk")
```

Comment:
Ideally this would be a relative path in the translations file.

Reply:
Totally. I must have thought that I was in a JS template. I fixed this in a newer commit.